src/Components/Signup.jsx

```diff
@@ -48,11 +49,10 @@ class Signup extends Component {
   
   render() {
     let signupForm
-    let welcomeMessage
     let errorMessage
 
     if (this.props.currentUser.isSignedIn) {
-      welcomeMessage = <p id="welcome-message">Hello {this.props.currentUser.attributes.name}</p>
+          signupForm = ( <PaymentForm /> )
```

Comment:
This is not a great practice, you should use 2 separate variables and toggle them conditionally instead.
It can be difficult for the future programmers to read your code if `signupForm` can be the signup and payment form as well.